Snowflake.Data.Tests/IntegrationTests/StructuredTypesIT.cs

```diff
@@ -0,0 +1,1471 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using NUnit.Framework;
+using Snowflake.Data.Client;
+using Snowflake.Data.Core;
+using Snowflake.Data.Core.Converter;
+using Snowflake.Data.Tests.Client;
+using Snowflake.Data.Tests.Util;
+
+namespace Snowflake.Data.Tests.IntegrationTests
+{
+    [TestFixture]
+    [IgnoreOnEnvIs("snowflake_cloud_env", new [] { "AZURE", "GCP" })]
+    public class StructuredTypesIT : SFBaseTest
+    {
+        private const string StructuredTypesTableName = "structured_types_tests";
+
+        [Test]
+        public void TestSelectInsertedStructuredTypeObject()
```

Comment:
This test and the next one are covered by `TestSelectNestedStructuredTypeObject`. Maybe we can remove them?

Reply:
OK, I removed it.